E2a-TRST/TSTP/Dataset/README.md

```diff
@@ -0,0 +1,41 @@
+## Variations
+The following occurrence variations need to be checked and certified in relation to the targeted entities and concept templates:
+
+- IfcRail.RAIL - *requires Body AdvancedSweptSolid Geometry*
+- IfcTrackElement.SLEEPR - *requires Mapped Geometry*
+- IfcRail.CHECKRAIL - *requies Body Tessallation Geometry*
+- IfcTrackElement.FROG - *requies Body Tessallation Geometry*
+- IfcMechanicalFastner.RAILFASTNING - *requires Mapped Geometry*
+
+## Model Dataset
+This test case utilises the attached dataset documented by the following drawings and data schedule. 
+
+*This is a later step that involved the detailed documentation of the certification dataset (model)*
+
+A track turnout panel is modelled as an instance of IfcElementAssembly with Predefined Type TURNOUTPANEL.
+It is decomposed by elements including IfcRail.RAIL, IfcTrackElement.SLEEPER, IfcRail.CHECKRAIL, IfcTrackElement.FROG, IfcMechnicalFastner.RAILFASTNING, IfcMechnicalFastner.RAILJOINT.
+All the placement of these elements should be based on the placement of the turnout panel, and the turnout panel as an assembly is placed based on the alignment.
+It is required to define the explicit and start and end stationing of the turnout panel.
```

Comment:
- typo: explicit *and* start?
- perhaps a mention that these should be `IfcReferent`s would be beneficial

Reply:
Updated, thanks